README.adoc

```diff
@@ -17,6 +17,130 @@ This repository follows the monorepo pattern and includes the following projects
+
+Above are the generated private and public keys.
+
+See Decidim docs at docs/services/bulletin_board.md in order to set them up.
+```
+
+You need then to add the private part to you Decidim Server as the ENV var `AUTHORITY_PRIVATE_KEY` (`AUTHORITY_PRIVATE_KEY={"kty":"RSA","n":"wbt...`). Copy verbatim, no break lines. Also, create a new name for you Decidim instance and place it in the ENV `AUTHORITY_NAME`. You will need this identifier to configure the bulletin board server. Other variables are important, see the [Decidim docs](https://docs.decidim.org/en/develop/services/elections_bulletin_board.html) for it.
+
+Then you need to add the public part to the bulletin board server. This is done in the **bulletin board server**. You can do it with the following command:
+
+```bash
+bin/rails 'client:add_authority[AuthorityName, public-key]
+```
+
+- **AuthorityName**; is the name of the authority, for instance, the name of the Decidim tenant. It can be anything as long is the same in Decidim and in the bulletin board.
+- **public-key**: is the public key generated in the Decidim server by the previously explained command.
+
+This command will produce an output like this:
+
+```
+$ bin/rails 'client:add_authority[AuthorityName,kty=RSA&n=wbt6n7uvR7O6KgOP_Idq-YzWf4DrjcpQSdQDCn0NB_nVmgDRq5kxe3z63Dpk2nWY8ZfZz5THCkxkz-BllnxuYMjfBHD4zpkn5V3OIMcTpJTHH9kiAOYFFvCKDHlx9GQ86d_U9bkmLq-ucKNhk-tdLZ0qv4FVJyPn6Q-hBnhkGC_lnGhe0vePVHE2nSGHPRhwqE4yFnSGj6L2o11oG2sQvFlM_a0xyTE-PX4P4quJUZo1I3-hQjPZqmniaL9L7bqBdAYkQLC8IEjLfBJC7GeXmRGkm45TmuKDy57JydhOvzLCFG4U6linqdv-8xkkejlzkYwFJc_R3WiuuFT65JmyiIQVVHFWPPl403bnxi5LB3DP9CT7zngd7tdJwSXzndBOpOoLTWbvrQTHhshw-XxPeP548rN2UDyzUtg1Rh__TmGExj3ZlXOqA1H6TPxmy4kjnqcOfgeH-8omWIq2Woa46EfHQ2BsLzqE6uYeiau22cv3SdPKLuZLQ3fnvKNwRmToJ44Q6aglkkOFoyFnEJdg3FlymddMNBMGEWTHC2kLJhlAHTpR0UPtPNsG2fVjDRoIWgBSpQ4-1LaZc3rxPFDKYermpmA6nImuUBaQ4VJ6KXExzBpcubOm8FWYYLWpiOJsReFDZwzI4uIzY1wC172MNhf4tF6ZJ-qJKeBMcIyVDa8&e=AQAB&kid=f18965a6a859e284f0ea076d2140e212f1bbd484e029b69c3d9cc06b34e61b0a]'
+
+Authority 'AuthorityName' successfuly added!
+The API key for this authority is: WbyrkuMrn7Pgk-ebKQpQzpSkKSv-T-r0TaclfMFMspV7g6qoFRqFwYmAnb0PZqATDZx3WdIf6SIF-f40s16wBTbdlST-3ZZ3wWzctyvdb4Wha_hranhHPG6sR2Qmq5uk
+```
+
+Note that an API key is returned, you will also need to add this API key as an ENV var in your Decidim server. This is used to ensure only that Decidim instance can communicate with the bulletin board.
+
+To summarize, these are the ENV vars you need to add to your Decidim server once all this process has finished:
+
+===== ENV vars for the Decidim Server
```

Comment:
(Not actionable, just commenting this to synchronize visions about how I feel this should work) 

I always thought that this whole messy process can be improved by actually changing the outputs and instructions to say things like: 

```console
Authority 'AuthorityName' successfuly added!

Add the following environment variable to the Decidim application configuration: 
BULLETIN_BOARD_API_KEY="WbyrkuMrn7Pgk-ebKQpQzpSkKSv-T-r0TaclfMFMspV7g6qoFRqFwYmAnb0PZqATDZx3WdIf6SIF-f40s16wBTbdlST-3ZZ3wWzctyvdb4Wha_hranhHPG6sR2Qmq5uk" 
```

Reply:
Sure, the only think that might be inconvenient on polluting command outputs with instructions is that it makes harder for automated scripts to work (goes against unix general philosophy of "silence is gold")
In any case, many other commands have instructions embeded